tests/Feature/Groups/GroupCreateTest.php

```diff
@@ -36,6 +39,12 @@ public function testCreate()
         self::assertEquals($group->name, $ret[0]['name']);
     }
 
+    public function testCreateGroupAsRestarter() {
```

Comment:
Noting that this one should start failing soon after https://therestartproject.atlassian.net/browse/RES-1867 is implemented, but will keep for now.